src/Form/Admin/CandidateManagedAreaType.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\Form\Admin;
+
+use App\Entity\Geo\Zone;
+use App\Entity\ManagedArea\CandidateManagedArea;
+use App\Repository\Geo\ZoneRepository;
+use Symfony\Bridge\Doctrine\Form\Type\EntityType;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\FormEvent;
+use Symfony\Component\Form\FormEvents;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class CandidateManagedAreaType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('zone', EntityType::class, [
+                'label' => false,
+                'required' => false,
+                'class' => Zone::class,
+                'query_builder' => function (ZoneRepository $repository) {
```

Comment:
it will load 2K entities for filling the <select>, maybe we can use an autocomplete?